airqo-platform/AirQo-frontend
# [NetManager] Add custom charts

#### Summary of Changes (What does this PR do?)
- Adds the more flexible custom charts ([apex charts](https://apexcharts.com/)) to netmanager
- The new charts are currently only available on `device management` and `device overview` pages

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch this branch locally
* Install requirements using `npm install`
* Start Application using `npm run stage-mac` (MacOs) or `npm run stage-pc` (Windows PC)
* Check the charts on `device management` and `device overview` pages and give feedback

#### What are the relevant tickets?
- [PLAT-615](https://airqoteam.atlassian.net/browse/PLAT-615)

#### Screenshots (optional)
![image](https://user-images.githubusercontent.com/39955305/121599419-af757300-ca4b-11eb-9f7b-05a9037c1446.png)
![image](https://user-images.githubusercontent.com/39955305/121599507-cf0c9b80-ca4b-11eb-9c8e-2c07cd2f83c4.png)
